src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
```

Comment:
- [ ] Please one method is missing and this is from the project requirement. Therefore, Imports `categoriesReducer` from `src/redux/categories`
- [ ] One more: Please Combine both reducers(`booksReducer` and `categoriesReducer`) into a root reducer by using `combineReducers` function. Kindly [see the project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md)